Formula/xcbuild.rb

```diff
@@ -0,0 +1,24 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git",
+    :tag => "0.1.0",
+    :revision => "08575f65f1c907e280ed4663b5461b7cfaf5cfaa",
+    :shallow => false
+  head "https://github.com/facebook/xcbuild.git",
+    :shallow => false
```

Comment:
Just to be confusing:
```ruby
head "https://github.com/facebook/xcbuild.git", :shallow => false
```
Was actually fine. We tend to drop `:tag` and `:revision` onto their own lines but it doesn't make as much sense for the `:shallow` modifier.